### DISCUSSION TITLE
find help

### DISCUSSION THREAD

**Discussion by wjycau** [PMEAL/OpenPNM#2904] [Q&A]
Hello Professor, I have encountered the following issue when applying PoreSpy to extract the soil pore network and visualize it. When visualizing spheres, there is no error. However, when I try to visualize throats based on their diameter, the "Tube" button in Paraview is grayed out, following the steps on the official website. Could you please explain why this is happening? Below is the result of my execution.
![6d8b59e94e6c869625fc9c0fe7e3a91](https://github.com/PM
![2499440b72322978b463cb180f7e8e6](https://github.com/PMEAL/OpenPNM/assets/151709787/494d5a99-9e38-48a6-8727-abe30c6c0b40)
![c67a8a7a7ec728a1f1fcf82bb022f12](https://github.com/PMEAL/OpenPNM/assets/151709787/23eb1635-5361-4a7c-9e94-3f8581b0b6e0)
![1afc060a62aae74d1dbbe1ac9ca7f76](https://github.com/PMEAL/OpenPNM/assets/151709787/b3b5d99e-28b4-442f-90df-6d97c91db9f1)

EAL/OpenPNM/assets/151709787/2c34b5eb-d027-413f-8924-74264dec552c)

**Comment by ma-sadeghi** [comment]
Hi,
To be able to help you, your question needs to be well defined:

1. Come up with a descriptive title
2. Describe the issue in your own words
3. Produce and show a minimum working example (MWE*)

* An MWE is a short script that reproduces the issue you're having. It's NOT necessarily the script you've been working on, but a small subset of it. You need to spend some time for producing a good quality MWE as it helps developers help you more efficiently.

** A screenshot is not an MWE.

**Comment by jgostick** [comment]
As you can tell by @ma-sadeghi's comment, your question is not well posed, and also fails the "Did I read the docs before asking the devs?" test.  The process for rendering throats as tubes is explained [here](https://openpnm.org/examples/tutorials/10_visualization_options.html).
